util/topgen/merge.py

```diff
@@ -574,6 +575,9 @@ def amend_pinmux_io(top):
         # Parse how many signals
         m = lib.get_module_by_name(top, mname)
 
+        if m is None:
+            log.error("Module {} is not searchable.".format(mname))
```

Comment:
You use `m` in the statement below; does it copy with `m` being `None`, or are you missing a return or else here?